src/Parser/Warrior/Arms/Modules/Features/SpellUsable.js

```diff
@@ -0,0 +1,36 @@
+import SPELLS from 'common/SPELLS';
+import CoreSpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+class SpellUsable extends CoreSpellUsable {
+  static dependencies = {
+    ...CoreSpellUsable.dependencies,
+  };
+
+  on_byPlayer_applybuff(event) {
+    if(super.on_byPlayer_applybuff) {
+      super.on_byPlayer_applybuff(event);
+    }
+    this.handleTacticianBuff(event);
+  }
+
+  on_byPlayer_refreshbuff(event) {
+    if(super.on_byPlayer_refreshbuff) {
+      super.on_byPlayer_refreshbuff(event);
+    }
+    this.handleTacticianBuff(event);
+  }
+
+  handleTacticianBuff(event) {
+    // If the buff being applied/refreshed is tactician, refresh the cooldowns of mortal strike and colossus smash.
+    if (SPELLS.TACTICIAN.id === event.ability.guid) {
```

Comment:
My bad for missing this the first time around but we prefer avoiding nested if statements. Instead check for the opposite and return early when possible.